exposure-notification/src/test/java/fi/thl/covid19/exposurenotification/efgs/FederationServiceSyncWithDaoTestIT.java

```diff
@@ -170,7 +182,7 @@ public void downloadRetryMaxLimit() {
 
     @Test
     public void downloadKeysTransmissionRiskLevel() {
-        List<TemporaryExposureKey> keys = FederationGatewayBatchUtil.transform(FederationGatewayBatchUtil.transform(
+        List<TemporaryExposureKey> keys = transform(transform(
```

Comment:
transform transform. For the autobots :D Seriously though, maybe those function names could describe what/how they transform sot this would make more sense.

Reply:
I don't know if it's the best convention but I've often settled to names like toX or X.from in transformations. The parameter-name, field name or function class then give the rest of the meaning.
For example (preferring the first one):
inputs = toInternalForm(externalFormat)
inputs = externalFormat.toMyInternalClass()
inputs = MyInternalClass.from(externalFormat)

Reply:
This just to for tests, that I don't need to manually handle autogenerated fields from transform. It's visible from methor definition what it will take as parameter and what you can except. Anyway, in actual code it is never used like that.

Reply:
Anyway, this is something which could be discussed more. I see your point. :)